examples/tests/human.nbt

```diff
@@ -20,19 +20,27 @@ assert_eq((1 day            -> human), "1 day")
 assert_eq((1.37 day         -> human), "1 day + 8 hours + 52 minutes + 48 seconds")
 
 assert_eq((1 week           -> human), "7 days")
-assert_eq((1.5 weeks        -> human), "10 days + 12 hours")
```

Comment:
Why is this test missing now?

Reply:
Because of https://github.com/sharkdp/numbat/issues/589 :sweat_smile: , but with your answer there, I think I could improve the code here to have it work too :slightly_smiling_face: .

Reply:
Yep, fixed in 89a141b